samples/sensor/grow_r502a/src/main.c

```diff
@@ -64,13 +119,29 @@ static void trigger_handler(const struct device *dev,
 		finger_enroll(dev);
 	} else {
 		template_count_get(dev);
-		finger_match(dev);
+		finger_find(dev);
 	}
 }
 
+static void read_fps_param(const struct device *dev)
+{
+	int ret = 0;
+	struct r502a_sys_param res;
+
+	ret = r502a_read_sys_param(dev,&res);
+	if (ret != 0) {
+		printk("r502a read system parameter failed %d\n", ret);
+		return;
+	}
+
+	printk("baud %d\n", res.baud);
+	printk("addr 0x%x\n", res.addr);
+	printk("lib_size %d\n", res.lib_size);
+	printk("data_pkt_size %d\n", res.data_pkt_size);
```

Comment:
not really blocking but I think it would be great to update the contents of https://docs.zephyrproject.org/latest/samples/sensor/grow_r502a/README.html  so that these changes are reflected in the output of the program (there might also be other verbiage worth updating in this page?)

Reply:
Updated. Thanks for this comment.